eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshAdminServer.java

```diff
@@ -68,25 +94,73 @@
+        this.adminWebHookConfigOperationManage = eventMeshTCPServer.getAdminWebHookConfigOperationManage();
 
-    public EventMeshAdminServer(int port, boolean useTLS,
-        EventMeshHTTPConfiguration eventMeshHttpConfiguration, ClientManageController clientManageController) {
-        super(port, useTLS, eventMeshHttpConfiguration);
-        this.clientManageController = clientManageController;
-        this.httpHandlerManager = clientManageController.getHttpHandlerManager();
     }
 
 
     @Override
     public void init() throws Exception {
         super.init("eventMesh-admin-http");
-        httpConnectionHandler = new HttpConnectionHandler();
+        registerHttpHandler();
+    }
+
+    private void registerHttpHandler() {
+        initHandler(new ShowClientHandler(eventMeshTCPServer));
+        initHandler(new ShowClientBySystemHandler(eventMeshTCPServer));
+        initHandler(new RejectAllClientHandler(eventMeshTCPServer));
+        initHandler(new RejectClientByIpPortHandler(eventMeshTCPServer));
+        initHandler(new RejectClientBySubSystemHandler(eventMeshTCPServer));
+        initHandler(new RedirectClientBySubSystemHandler(eventMeshTCPServer));
+        initHandler(new RedirectClientByPathHandler(eventMeshTCPServer));
+        initHandler(new RedirectClientByIpPortHandler(eventMeshTCPServer));
+        initHandler(new ShowListenClientByTopicHandler(eventMeshTCPServer));
+        initHandler(new QueryRecommendEventMeshHandler(eventMeshTCPServer));
+        initHandler(new TCPClientHandler(eventMeshTCPServer));
+        initHandler(new HTTPClientHandler(eventMeshHTTPServer));
+        initHandler(new GrpcClientHandler(eventMeshGrpcServer));
+        initHandler(new ConfigurationHandler(
+            eventMeshTCPServer.getEventMeshTCPConfiguration(),
+            eventMeshHTTPServer.getEventMeshHttpConfiguration(),
+            eventMeshGrpcServer.getEventMeshGrpcConfiguration()));
+        initHandler(new MetricsHandler(eventMeshHTTPServer, eventMeshTCPServer));
+        initHandler(new TopicHandler(eventMeshTCPServer.getEventMeshTCPConfiguration().getEventMeshStoragePluginType()));
+        initHandler(new EventHandler(eventMeshTCPServer.getEventMeshTCPConfiguration().getEventMeshStoragePluginType()));
+        initHandler(new MetaHandler(eventMeshMetaStorage));
+        if (Objects.nonNull(adminWebHookConfigOperationManage.getWebHookConfigOperation())) {
+            WebHookConfigOperation webHookConfigOperation = adminWebHookConfigOperationManage.getWebHookConfigOperation();
+            initHandler(new InsertWebHookConfigHandler(webHookConfigOperation));
+            initHandler(new UpdateWebHookConfigHandler(webHookConfigOperation));
+            initHandler(new DeleteWebHookConfigHandler(webHookConfigOperation));
+            initHandler(new QueryWebHookConfigByIdHandler(webHookConfigOperation));
+            initHandler(new QueryWebHookConfigByManufacturerHandler(webHookConfigOperation));
+        }
+    }
```

Comment:
How about putting this method (registering endpoints) in a Manager class just like the ConsumerManager does in `EventMeshHTTPServer`?

Reply:
Sure,please review.